fpdf/fpdf.py

```diff
@@ -3190,13 +3195,11 @@ def _parse_chars(self, txt):
             )
         return tuple(fragments)
 
-    def _get_fallback_font(self, char, style=""):
-        "returns which fallback font has the requested glyph"
+    def _get_fallback_font(self, char):
+        "Returns which fallback font has the requested glyph"
         if not self._fallback_font_ids:
             return None
         for font in self._fallback_font_ids:
```

Comment:
This just returns the first font found that happens to cover the requested character.
Shouldn't we try to use a "best match" strategy instead?

For example, if the original font is bold, and we have both a regular and a bold fallback available covering it, then whichever happens to be listed first will be picked, even if a better match is theoretically available.